src/assets/illustrations/fallback/fallback_code.svg

```diff
@@ -0,0 +1,81 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="240" height="156" viewBox="0 0 190 106">
```

Comment:
Can we prepare these images at 800 by 450 a la our other article illustrations? (This feedback applies to all images.)